Connection/Connection.php

```diff
@@ -90,6 +97,9 @@ public function getMailbox(): Mailbox
     
     public function testConnection(bool $throwExceptions = false): bool
     {
+        if(!$this->isEnabled()) {
```

Comment:
Please add space between if and "(" (see: Symfony Coding Standard https://symfony.com/doc/current/contributing/code/standards.html#symfony-coding-standards-in-detail)